contribs/gmf/src/services/objecteditingquery.js

```diff
@@ -220,3 +220,17 @@ gmf.ObjectEditingQuery.GetQueryableLayerNodesResponse;
  * }}
  */
 gmf.ObjectEditingQuery.QueryableLayerInfo;
+
```

Comment:
`@typedef` --> don't need to.

`@struct / @consrtructor` requires it.

I think... Thoughts?